crates/services/relayer/src/service.rs

```diff
@@ -345,7 +345,7 @@ where
                 Err(anyhow::anyhow!("The relayer got a stop signal"))
             },
             block = self.eth_node.get_block(ethers_core::types::BlockNumber::Finalized) => {
-                let block_number = block.map_err(anyhow::Error::msg)?
+                let block_number = block.inspect_err(|err| tracing::error!(?err, "eth provider failed to get block")).map_err(anyhow::Error::msg)?
```

Comment:
Either way here.

Reply:
Changed in https://github.com/FuelLabs/fuel-core/pull/3004/commits/7a52b5039f6ad02b18db4ca30bec4f5d61726424